gemrb/GUIScripts/LUSpellSelection.py

```diff
@@ -246,7 +246,7 @@ def OpenSpellsWindow (actor, table, level, diff, kit=0, gen=0, recommend=True, b
 		if recommend:
 			SpellsWindow.Focus()
 		else:
-			SpellsWindow.ShowModal (MODAL_SHADOW_NONE)
+			SpellsWindow.ShowModal (MODAL_SHADOW_GRAY)
```

Comment:
this is fine, the others will always pick the other branch and iwd1 is still using its own code.

Reply:
Thanks!